graphql/queryResolvers/allUsers.test.js

```diff
@@ -0,0 +1,33 @@
+import { allUsers } from './allUsers'
+import db from '../../helpers/dbload'
+
+const mockUsers = [
+  {
+    username: 'lolcakes',
+    id: 2
+  },
+  {
+    username: 'superLolcakes',
+    id: 6
+  }
+]
+
+const ctx = {
+  req: {
+    user: { isAdmin: 'true' }
+  }
+}
+
+describe('allUsers resolver', () => {
+  const { User } = db
+
+  test('should return list of users', async () => {
+    User.findAll = jest.fn().mockReturnValue(mockUsers)
+    expect(allUsers(null, null, ctx)).toEqual(mockUsers)
+  })
+
+  test('Should throw Error when user is not an admin when querying allUsers', () => {
```

Comment:
I see that the resolver returns `null` when the `isAdmin` function returns a falsey value. But where is the Error thrown?

Reply:
You are right, let me update that test description. Thank you!